src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -496,6 +503,19 @@ public YesNo getRegulation35Selection() {
         }
     }
 
+    @JsonIgnore
+    public YesNo getSchedule9Paragraph4Selection() {
+        if (sscsUcCaseData != null) {
```

Comment:
could call the getter so that there we could remove this null check

Reply:
Much better - thanks!